build.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+set -e
+path=$(dirname "$0")
+node build.js "$path/tests" > "$path/cts.json"
```

Comment:
I don't know much about node. How heavy a dependency is it, in terms of size and frequency of required upgrade?

Reply:
I can't see nodejs being a substantial obstacle for anyone wishing to mess with the CTS. It's available for pretty much every popular OS and the distribution size is about 30 MB. https://nodejs.org/en/download

Reply:
My idea was that the build script would run on the server, so you wouldn't need to install node locally. You would just make changes to the files in `tests/` directory and submit a PR. GitHub actions would then take care of validating the changes and updating the `cts.json`.

Reply:
I'm not sure that's the fastest way to get feedback if someone has an implementation and is working on the CTS. I suspect they'd install node and run build.sh to test their changes locally.